crates/pallet-domains/src/lib.rs

```diff
@@ -599,12 +606,12 @@ mod pallet {
     pub(super) type HeadReceiptNumber<T: Config> =
         StorageMap<_, Identity, DomainId, DomainBlockNumberFor<T>, ValueQuery>;
 
-    /// Whether the head receipt have extended in the current consensus block
+    /// The hash of the new head receipt added in the current consensus block
     ///
     /// Temporary storage only exist during block execution
     #[pallet::storage]
-    pub(super) type HeadReceiptExtended<T: Config> =
-        StorageMap<_, Identity, DomainId, bool, ValueQuery>;
+    pub(super) type NewAddedHeadReceipt<T: Config> =
```

Comment:
shouldn't this change require a migration?

Reply:
Not necessary because it is a temporary storage that only exists during block execution and is cleared at block finalization.